src/Animate.js

```diff
@@ -125,6 +126,10 @@ class Animate extends PureComponent {
     if (this.stopJSAnimation) {
       this.stopJSAnimation();
     }
+
+    if (onAnimationEnd) {
+      onAnimationEnd();
+    }
```

Comment:
Nit: Conditional invocation is more concise.
```suggestion
    this.onAnimationEnd?.();
```

Reply:
Could also refactor the others in the same manner.

Reply:
Since this is how these checks exist currently im going to leave it. If we come back and refactor at all we'll touch these

Reply:
Totally fine, that is why it is only a nit :D